specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3568 @@
+          },
+          {
+            "in": "query",
+            "name": "maxpagesize",
+            "description": "Number of objects to return per page",
+            "type": "integer",
+            "format": "int32",
+            "default": 20
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/JobCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets a list of jobs with pagination (max page size)": {
+            "$ref": "./examples/Jobs_GetJobsWithPageSize.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/routing/jobs/{id}/position": {
+      "get": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Gets a job's position details.",
+        "operationId": "JobRouter_GetInQueuePosition",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Id of the job",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/JobPositionDetails"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets a job position in a queue": {
+            "$ref": "./examples/Jobs_GetInQueuePosition.json"
+          }
+        }
+      }
+    },
+    "/routing/workers/{workerId}/offers/{offerId}:accept": {
+      "post": {
+        "tags": [
+          "Offers"
+        ],
+        "summary": "Accepts an offer to work on a job and returns a 409/Conflict if another agent accepted the job already.",
+        "operationId": "JobRouter_AcceptJobAction",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "offerId",
+            "description": "Id of the offer",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "workerId",
+            "description": "Id of the worker",
+            "required": true,
+            "type": "string"
```

Comment:
Path parameter "workerId" should appear before "offerId" (to match order in path).